ports/stm32/eth.c

```diff
@@ -405,6 +465,11 @@ STATIC int eth_mac_init(eth_t *self) {
         eth_dma.tx_descr[i].tdes1 = 0;
         eth_dma.tx_descr[i].tdes2 = TX_BUF_SIZE & TX_DESCR_2_B1L_Msk;
         eth_dma.tx_descr[i].tdes3 = 0;
+        #elif defined(STM32H5)
```

Comment:
It looks like this code is exactly the same as the H7 version above.  Can it just use the same code with `#if defined(STM32H5) || defined(STM32H7)`?

Reply:
Indeed most of the ETH code is identical for H5 and H7. I kept it separate so that I could more easily implement and test the fate.

Could you give the Ethernet implementation a try? If you or someone could confirm the code is ok, I can merge the sections together.

Reply:
Done